src/Microsoft.ML.Data/Transforms/ValueToKeyMappingTransformer.cs

```diff
@@ -178,13 +171,41 @@ public ColumnInfo(string input, string output, int maxNumTerms = ValueToKeyMappi
             public readonly bool TextKeyValues;
 
             protected internal string Terms { get; set; }
-        }
 
-        public const string Summary = "Converts input values (words, numbers, etc.) to index in a dictionary.";
-        public const string UserName = "Term Transform";
-        public const string LoaderSignature = "TermTransform";
-
-        public const string FriendlyName = "To Key";
+            /// <summary>
+            /// Describes how the transformer handles one column pair.
+            /// </summary>
+            /// <param name="input">Name of input column.</param>
+            /// <param name="output">Name of the column resulting from the transformation of <paramref name="input"/>. Null means <paramref name="input"/> is replaced.</param>
+            /// <param name="maxNumTerms">Maximum number of terms to keep per column when auto-training.</param>
+            /// <param name="sort">How items should be ordered when vectorized. By default, they will be in the order encountered.
+            /// If by value items are sorted according to their default comparison, for example, text sorting will be case sensitive (for example, 'A' then 'Z' then 'a').</param>
```

Comment:
>by value [](start = 19, length = 8)

What does "by value items" mean?

Reply:
Here and in other xml docs.

---
In reply to: [240377704](https://github.com/dotnet/machinelearning/pull/1811#discussion_r240377704) [](ancestors = 240377704)

Reply:
Are you confused by "by" or in general value items?

---
In reply to: [241128087](https://github.com/dotnet/machinelearning/pull/1811#discussion_r241128087) [](ancestors = 241128087,240385251,240377704)

Reply:
Comma was missed. I added actual enum values for better understanding, is it better now?

---
In reply to: [241132520](https://github.com/dotnet/machinelearning/pull/1811#discussion_r241132520) [](ancestors = 241132520,241132122,241130103,241128087,240385251,240377704)